tests/integration/models/reddit/test_comment.py

```diff
@@ -182,6 +182,21 @@ def test_refresh__removed_comment(self):
         assert ('This comment does not appear to be in the '
                 'comment tree',) == excinfo.value.args
 
+    def test_refresh__with_reply_sort_and_limit(self):
+        with self.recorder.use_cassette('TestComment.test_refresh__with_reply_sort_and_limit'):
+            comment = Comment(self.reddit, 'e4j4830')
+            comment.reply_limit = 4
+            comment.reply_sort = 'new'
+            comment.refresh()
+            replies = comment.replies
+        lastCreated = float('inf')
+        for reply in replies:
+            if type(reply) is Comment:
```

Comment:
Can you use `isinstance(reply, Comment)` here?

Reply:
Can do!